db/migrate/20231003182930_add_species2_pets.rb

```diff
@@ -1,5 +1,5 @@
 class AddSpecies2Pets < ActiveRecord::Migration[7.0]
   def change
-    add_column :pets, :species, :integer, null: false
```

Comment:
This is an existing migration right? I though that this was already in `main`

Reply:
this is true... how the heck did that get added agian?